tests/baselines/reference/typeFromPropertyAssignment12.types

```diff
@@ -30,7 +30,7 @@ var pos = new Outer.Pos(1, 'x');
 >pos : Pos
 >new Outer.Pos(1, 'x') : Pos
 >Outer.Pos : typeof Pos
->Outer : { (element: any, config: any): void; Pos(line: any, ch: any): void; }
+>Outer : { (element: any, config: any): void; }

```

Comment:
You're right, this is a result of the change in `resolveAnonymousTypeMembers`. And it breaks intelllisense -- you get no suggestions for `Outer.`. I haven't figured out why though.